docs/modules/ROOT/pages/tutorials/tutorial.adoc

```diff
@@ -227,7 +227,7 @@ To restore using Restic, set these variables (in a Unix-based system; for Window
 [source,bash]
 ....
 export KUBECONFIG=""
-export RESTIC_REPOSITORY=s3:$(minikube service minio --url)/backups/
+export RESTIC_REPOSITORY=s3:$(minikube service list | grep 9000 | cut -d"|" -f 5 | tr -d "[:space:]")/backups/
```

Comment:
Is there no other option to get the output? e.g. `-o json` and with jq?

Reply:
unfortunately it does not look like: https://minikube.sigs.k8s.io/docs/commands/service/#minikube-service-list
It's a bit hacky and if they decide to add another column it will break again, but do get it running for the tutorial "good enough for now" workaround.

Reply:
What exactly is the output of `minikube service list`? Can you post a sample?
I might be able to come up with a `kubectl` equivalent that is more stable and less prone to changing output of minikube

Reply:
``` 
$ minikube service list
|-------------|-------------------------|--------------|---------------------------|
|  NAMESPACE  |          NAME           | TARGET PORT  |            URL            |
|-------------|-------------------------|--------------|---------------------------|
| default     | k8up-1652941675-metrics | No node port |
| default     | kubernetes              | No node port |
| default     | mariadb                 | No node port |
| default     | minio                   | api/9000     | http://192.168.49.2:30362 |
|             |                         | webui/9001   | http://192.168.49.2:32176 |
| default     | wordpress               |           80 | http://192.168.49.2:31705 |
| kube-system | kube-dns                | No node port |
|-------------|-------------------------|--------------|---------------------------|
``` 

Could be difficult:
``` 
kubectl describe service minio 
Name:                     minio
Namespace:                default
Labels:                   <none>
Annotations:              <none>
Selector:                 app=minio
Type:                     NodePort
IP Family Policy:         SingleStack
IP Families:              IPv4
IP:                       10.109.69.127
IPs:                      10.109.69.127
Port:                     api  9000/TCP
TargetPort:               9000/TCP
NodePort:                 api  30362/TCP
Endpoints:                172.17.0.6:9000
Port:                     webui  9001/TCP
TargetPort:               9001/TCP
NodePort:                 webui  32176/TCP
Endpoints:                172.17.0.6:9001
Session Affinity:         None
External Traffic Policy:  Cluster
Events:                   <none>

$ kubectl get service minio 
NAME    TYPE       CLUSTER-IP      EXTERNAL-IP   PORT(S)                         AGE
minio   NodePort   10.109.69.127   <none>        9000:30362/TCP,9001:32176/TCP   10m
```

I'm not sure  how to find out the IP address 192.168.49.2 for minikube cluster

Reply:
A kubectl port-forward would probably be easier... Also would decouple it a bit from minikube.

`kubectl port-forward svc/minio 9001:9001`

Then the gui should be available under `http://localhost:9001`

Reply:
updated documentation and tested quickly: 
https://github.com/k8up-io/k8up/pull/666/commits/dfbb2f14d9c478e8eb890c7adeefbffb10d06e57